### ISSUE TITLE
Annex B advantages & superiorities

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/dggs-4-axis-aligned#8]
In Annex B before diving into the GEOSOT description, a general overview of AA DGGS is presented.
In there, it seems to claim that axis-aligned has all the advantages of DGGS and superiorities:

> On the basis of meeting all the advantages of DGGS, AA DGGS embodies the following superiorities:

While I am myself a proponent of the added simplicity of axis-aligned DGGS, this should probably be better balanced to acknowledge that traditional DGGS with equal area cells do have some advantages, such as making area/volume calculations much easier once data is mapped to the DGGS by simply counting cells, while AA DGGS have to apply a correction factor for each cell.

Also, since this annex is focused on the GEOSOT description, perhaps this overall introduction of axis-aligned DGGS belongs elsewhere in the introduction of the document?

**Comment by fuhuren** [comment]
Yes, equal area DGGS has the simplicity in calculating the size of cells, while AA DGGS has the simplicity in mapping data in to cells and understanding the exact position of them. 

We will add a general introduction of AA DGGS as required. Thanks for comment.
